hermes-common/src/test/groovy/pl/allegro/tech/hermes/common/metric/counter/zookeeper/CounterMatcherTest.groovy

```diff
@@ -34,6 +34,22 @@ class CounterMatcherTest extends Specification {
         subscriptionName == "subscription"
     }
 
+    def "should match fancy subscription delivered"() {
```

Comment:
what does "fancy" mean? would be great to have precise name for this test

Reply:
ok